src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -508,6 +508,8 @@ private static LibraryDependency ToPackageLibraryDependency(IVsReferenceItem ite
 
                 // Mark packages coming from the SDK as AutoReferenced
                 AutoReferenced = GetPropertyBoolOrFalse(item, "IsImplicitlyDefined"),
+
+                GeneratePathProperty = GetPropertyBoolOrFalse(item, "GeneratePathProperty")
```

Comment:
Can you also create NET core sdk team to pass this new property `GeneratePathProperty` in nomination?

Reply:
As you mentioned this is a metadata, so you can ignore this comment.